2020-01-15-standup-meeting.md

```diff
@@ -0,0 +1,91 @@
+---
+breaks:  false # See: https://github.com/hackmdio/codimd/issues/40#issuecomment-172927690
+private: false # See: https://github.com/hyphacoop/hyphacoop-chatbot#archive
+---
+# 2020-01-15 Hypha Worker Co-op: Standup Meeting
+
+<sup>[from template][standup-template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
+
+Time:       Wed, Jan 15, 5:00-5:30 pm  
+Location:   https://jitsi.hypha.coop/hyphacoop  
+Attending:  uv, pc, gi, el, bl, dc, later: yj  
+Cultivator: uv  
+Notetaker:  pc :raising_hand: Next up: yj, uv, gi, el, dc, bl, pc
+
+## Agenda
+
+- Standup (3 min/person)
+  1. What did you work on this past week?
+  2. What do you plan to work on this coming week?
+  3. What are your blockers?
+
+## Standup Notes
+
+- what was your first email handle?
+    - ben: ben_11991001 (ICQ NUMBA 4LYFE)
+    - udit: udit10@ (age)
+    - garry: antixt@ (script kiddie)
+    - elon: SAMESIES as now. COMMITMENT
+    - dc: used her a/s/l. burner addresses every yeeeeaaar. then greek mythos.
+
+
+- Person: ben
+    - last: lots of docs. wrote dweb proposal.
+    - next: dweb call is next week. call with Wendy on **Jan 21**
+    - blockers: #todo needs feedback on  https://github.com/hyphacoop/organizing-private/pull/11
+
+- Person: elon
+    - last: infra meeting.
+    - next: cultivating infra.
+    - blockers: none.
+
+- Person: garry
+    - last: n/a
+    - next: catchup. branding work.
+        - udit: bizdev meeting wants to make easier for garry w/ limited time.
+    - blockers: feels in need of recap.
+
+- Person: dc
+    - last: governance stuffs. cultivated gov wg! bizdev stuff. reviewed PRs. insurance call.
+    - next: more governance stuff re: objectives w goal-planning.
+    - blockers: none.
+
+- Person: patcon
+    - last: recently updated tix, updated crm deals, emails (code for canada, bookkeeper), payroll was run, learning accounting software, wrote guide doc for storing sensitive information, email address policy, circle-ci deploy fixes in handbook
+    - next: figuring out personal emails (ben feels priority), getting bookkeeper sorted, push some bizdev activities forward (CfC or CWCF grant)
+    - blockers: none.
+
+- Person: udit
+    - last: cultivation, biz planning, worked on intro deck. liking the momentum
+    - next: discuss intro deck feedback tomorrow
+    - blockers: no blocks
+
+- Person: yj
+    - last: infra & finance meetings, late-night payroll party
+    - next: not much
+    - blockers: no blocks
+
+## Discussions
+
+- dweb. ben will be out that way for 6 wks. july 16.
+- dc: #todo request for two questions. AGM stuff. by next week.
+- dc: when AGM will happen. June 30 year-end. AGM must be before, to have bylaws. we need a time we can all be here. hold 1.5 days: AGM and funtimes and strategy.
+    - ben: in Toronto all days of May
+- uv: i need bios from everyone #todo
+
+## Chat Summary
```

Comment:
```suggestion

```
Would remove this because it doesn't make sense in flow. Also when did this get added as a heading?  I thought we were keeping video call chats as an ephemeral space and am not super comfy with all my chat logs going up on here